### ISSUE TITLE
Could I share your validation_set.mat for thumos'14 dataset?

### ISSUE THREAD

**Issue by junmuzi** [VisionLearningGroup/R-C3D#49]
Hi @huijuan88,

Could I share your validation_set.mat for thumos'14 dataset? Thank you very much.

META_FILE='${PATH}/data/val/validation_set_meta/validation_set.mat'

**Comment by huijuan88** [comment]
You can get it from the thumos’14 dataset annotation.




On Jan 10, 2019, at 06:33, junmuzi <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi @huijuan88<https://github.com/huijuan88>,

Could I share your validation_set.mat for thumos'14 dataset? Thank you very much.

META_FILE='${PATH}/data/val/validation_set_meta/validation_set.mat'

**Comment by junmuzi** [comment]
@huijuan88, 

As you say "we divide 200 untrimmed videos from the validation set into 180 training and 20 held out videos to get the best hyperparameter setting." in your paper, so I want to know which 20 videos are used for validation? Could I share these video names?  Thank you very much.
